# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/mathml/presentation-markup/operators/operator-dictionary-spacing-004.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/mathml/presentation-markup/operators/operator-dictionary-spacing-004.html.ini
--- a/testing/web-platform/meta/mathml/presentation-markup/operators/operator-dictionary-spacing-004.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/mathml/presentation-markup/operators/operator-dictionary-spacing-004.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -7,11 +7,11 @@
 
   [Operator dictionary chunk 3 - lspace/rspace]
     expected:
-      if (os == "linux") and not fission and not swgl and not debug and not webrender and (processor == "x86"): ["PASS", "FAIL"]
-      if (os == "linux") and not fission and not swgl and not debug and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
-      if (os == "linux") and not fission and not swgl and not debug and webrender: ["PASS", "FAIL"]
-      if (os == "linux") and not fission and not swgl and debug: ["PASS", "FAIL"]
-      if (os == "linux") and not fission and swgl: ["FAIL", "PASS"]
+      if (os == "linux") and not fission and (processor == "x86_64") and webrender and debug: ["FAIL", "PASS"]
+      if (os == "linux") and not fission and (processor == "x86_64") and webrender and not debug: ["PASS", "FAIL"]
+      if (os == "linux") and not fission and (processor == "x86_64") and not webrender and debug: ["PASS", "FAIL"]
+      if (os == "linux") and not fission and (processor == "x86_64") and not webrender and not debug: ["FAIL", "PASS"]
+      if (os == "linux") and not fission and (processor == "x86"): ["PASS", "FAIL"]
       if (os == "linux") and fission: ["PASS", "FAIL"]
       if os == "mac": ["PASS", "FAIL"]
 